test/test2.js

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
- Please use meaningful names (no names like `test2` and so on)
 - The best practice is storing different types of tests in different directories (`test/e2e/*`, `test/api/*`)
 - Please fix indentation and other code style issues (use only `import`, spaces after commas, newlines before multiline declarations)